Packs/ApiModules/Scripts/MicrosoftAzureStorageApiModule/MicrosoftAzureStorageApiModule.yml

```
@@ -0,0 +1,15 @@
+commonfields:
+  id: MicrosoftAzureStorageApiModule
+  version: -1
+name: MicrosoftAzureStorageApiModule
+script: ''
+type: python
+subtype: python3
+tags:
```

should fix the validate issue
```suggestion
tags: []
```